lib/morph/database.rb

```diff
@@ -163,16 +163,16 @@ def write_sqlite_database(content)
     end
 
     def standardise_table_name(table_name)
-      sql_query_safe("ALTER TABLE '#{table_name}' " \
-        "RENAME TO '#{Database.sqlite_table_name}'", false)
+      sql_query_safe("ALTER TABLE \"#{table_name}\" " \
+        "RENAME TO \"#{Database.sqlite_table_name}\"", false)
     end
 
     def select_first_ten(table = table_names.first)
-      "select * from '#{table}' limit 10"
+      "select * from \"#{table}\" limit 10"
```

Comment:
While we're making this change what do you think about also using the convention of capitalising the keywords?